app/scripts/lib/ComposableObservableStore.js

```diff
@@ -72,7 +73,10 @@ export default class ComposableObservableStore extends ObservableStore {
           },
         );
       }
+
+      initialState[key] = store.state || store.getState?.();
```

Comment:
```suggestion
      initialState[key] = store.state ?? store.getState?.();
```
Should this be a nullish coalescing operator instead?

Reply:
Good point, yes I think it should. I don't think we have any examples of stores with a top-level falsy state, but I see no reason to assume that won't happen.

Reply:
Test added here: f3b82b45534b1c4e0ed6a432f2d42179cf39aed7